apps/client/app/components/selectList.tsx

```diff
@@ -0,0 +1,25 @@
+"use client"
+import React from 'react'
+
+interface SelectProps {
+    list: KeyValuePair[];
+    selected: string;
+}
+
+export default function SelectList({list, selected = ''}: SelectProps) {
+    const handleSort = (value: string) => {
+        console.log(value)
+        window.location.href = `http://localhost:4200/purchase-orders?sort=${value}`
+    }
```

Comment:
With a larger project it would be good to add all url routes to a const. This would allow for updating and not entering the wrong urls. Better for readability as well. 
```suggestion
    }
    const urlRoutes = {
        purchaseOrders: 'http://localhost:4200/purchase-orders',
        parentItems: 'http://localhost:4200/parent-items',
    }
```